tools/biddersettings.html

```diff
@@ -95,7 +94,7 @@
             inputObj.value = "var bidderSettings = " + bidSetStr;
 
             bidSetObj = {
-                bidderSpec: { bids: ["// Placeholder - Replace this with your bids array"] },
+                bidderSpec: { bids: ["// Placeholder - Your customized bidders array would go here"] },
```

Comment:
I would change the comment to "// Placeholder - Your customized bidders array may go here".
The options fields may be added to the JSON options object in any order.